run.js

```diff
@@ -38,22 +38,21 @@ const createNewContext = () => {
 const inspect = (val) => {
   try {
     return util.inspect(val, {
+      depth: 3,
       maxArrayLength: 20,
-      breakLength: 9999,
+      breakLength: Infinity,
       colors: false,
-      compact: false,
+      compact: true,
```

Comment:
depth/compact should default to what they already were. you can add another option for a "compact" mode or something.

Reply:
@devsnek the question is do you prefer this way to pass inputs, and options using env vars?
just yes or no, I don't mind closing this PR, just would like to avoid letting this hang forever